README.md

```diff
@@ -20,6 +20,25 @@ serve SSH and HTTPS on the same port.
 Compile and install
 ===================
 
+Docker container
+----------------
+
+For easy installation and usage you can use unofficial docker container.
+Container built with `libconfig`, `libwrap` and `libcap` support. 
+It can be found on [Docker Hub](https://hub.docker.com/r/riftbit/sslh/)
+
+**Example with --net host**
+
+```
+docker run --name sslh -d --env SSLH_OPTS='-p 0.0.0.0:443 --anyprot 192.168.0.1:8443' --net host --privileged --restart always riftbit/sslh
+```
+
+**Example with exposed ports**
+
```

Comment:
also, how does this interact with an apache and sshd?do these need to be in dockers themselves? do you just need to move  apache off 443? do you need to set sshd to 8443?

Reply:
no, apache or sshd can be installed as you wish (in container or not). If you run apache in host system, you can use container gateway ip, for example you run sslh in container with ip 172.17.0.2, apache installed in main system and have port 8443 - you should set 172.17.0.1:8443 in sslh config for apache. sorry for my bad english, i hope you understand me :)